src/utils.test.js

```diff
@@ -391,11 +391,18 @@ describe('utils', () => {
   });
 
   describe('logAxiosRetryError', () => {
-    let logEntry;
-    const addLogEntry = jest.fn().mockImplementation((obj) => (logEntry = obj));
-
     it('should log error correctly for signup', () => {
       // Arrange
+      let logEntry;
+      const addLogEntry = jest.fn().mockImplementation((obj) => (logEntry = obj));
```

Comment:
Just to know, why do you need to move it inside the _it_?

Reply:
For some reason the test broke otherwise. 
![image](https://user-images.githubusercontent.com/2439363/106293056-52abb580-622c-11eb-9044-3ae5efb84a38.png)

Reply:
I do not understand how it works, but I like the new implementation more than the previous one :)